run_tests.py

```diff
@@ -7,12 +7,7 @@
     os.environ['APP_SETTINGS'] = 'TestConfig'
 
     # NB: the following are set to avoid MissingEnvironmentVariable being raised in config.py
-    os.environ['JWT_SECRET'] = 'testsecret'
```

Comment:
Not a problem per se, but this block was useful when getting this running as this repo doesn't hint at any sensible defaults for these.  Could you add something somewhere in this repo to let new devs get started (maybe in the readme)?